data/config/schema.xml

```diff
@@ -75,6 +75,16 @@ to save the current settings to XML.
 		<short>Playlist screen timeout</short>
 		<long>How long it will take before the next song in the playlist starts automatically.</long>
 	</entry>
+	<entry name="game/results_timeout" type="int" value="5">
```

Comment:
shouldn't we make a seperate issue for the settings menu? it's becoming pretty cluttered right now, marbe rework the categories?

Reply:
Yes we should and I'll get on that when I finish this.